ide/docker.api/src/org/netbeans/modules/docker/api/DockerImage.java

```diff
@@ -88,6 +88,10 @@ public long getSize() {
         return size;
     }
 
+    /**
+     * @deprecated Removed from Docker Engine in v1.44. Return size value.
+     * @return 
+     */
     public long getVirtualSize() {
```

Comment:
In addition to the `@deprecated` javadoc tag please also add the `@Deprecated` annotation (see: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Deprecated.html)